content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -763,4 +763,8 @@ Delaying volume binding allows the scheduler to consider all of a pod's
 scheduling constraints when choosing an appropriate PersistentVolume for a
 PersistentVolumeClaim.
 
+### Note
```

Comment:
Should we also add a section explaining what a default storage class is and how to set one (if the provider didn't already)?

Reply:
@msau42 I've added a section to explain default storage class and pointed to another page on how to set the default storage class.